subprojects/packagefiles/libopenjp2/meson.build

```diff
@@ -20,22 +20,17 @@ openjpeg_version_build = '1'
 openjpeg_package_version = '@0@.@1@.@2@'.format(openjpeg_version_major, openjpeg_version_minor, openjpeg_version_build)
 
 
-m_dep = cc.find_library('m')
+m_dep = cc.find_library('m', required: false)
 thread_dep = dependency('threads')
 
-
 # clock_gettime might require -rt, or it might not. find out
-if not cc.has_function('clock_gettime', prefix : '#define _GNU_SOURCE\n#include <time.h>')
-  # XXX: untested
-  rt_dep = cc.find_library('rt')
-else
-  rt_dep = []
+librt = cc.find_library('rt', required : false)
+if librt.found() and cc.has_header_symbol('time.h', 'clock_gettime')
+  add_project_arguments('-DHAVE_LIBRT=1', language : 'c')
```

Comment:
Are you sure `HAVE_LIBRT` is used? The only place I saw for `clock_gettime()` is there:

https://github.com/uclouvain/openjpeg/blob/15615751b840d5b85a46292cf8fe393dab8e1a8b/src/bin/jp2/opj_decompress.c#L992

So it's only on linux, I think `librt = cc.find_library('rt', required : false)` should be enough here, no?